crates/node/src/shell/governance.rs

```diff
@@ -122,14 +122,6 @@ where
             TallyResult::Passed => {
                 let proposal_event = match proposal_type {
                     ProposalType::Default => {
-                        let proposal_code =
-                            gov_api::get_proposal_code(&shell.state, id)?
-                                .unwrap_or_default();
-                        let _result = execute_default_proposal(
-                            shell,
-                            id,
-                            proposal_code.clone(),
-                        )?;
```

Comment:
@grarco I think this is not needed in this branch right?

Reply:
Correct and I don't know why it is still there. I swear we talked about this thing already a few months ago and we fixed it.

Anyway, I also see that `execute_pgf_funding_proposal` return a boolean that we never use so we might remove that too?

Reply:
im now using the result to improve the print statement

Reply:
But it looks to me that `execute_pgf_funding_proposal` only ever returns `Ok(true)`